src/lib/storage/table.hpp

```diff
@@ -135,7 +136,7 @@ class Table : private Noncopyable {
       size_t current_size = chunk->size();
       row_counter += current_size;
       if (row_counter > row_number) {
-        return get<T>(
+        return boost::get<T>(
```

Comment:
@Bensk1 and I spend a while on Thursday trying to understand why a similar thing worked in student code :D